test/cli/config-cwd-case-insensitive/_config.js

```diff
@@ -0,0 +1,13 @@
+var os = require( 'os' );
+
+function toggleCase ( s ) {
+  return ( s == s.toLowerCase() ) ? s.toUpperCase() : s.toLowerCase();
+}
+
+module.exports = {
+  skip: os.platform() !== 'win32',
```

Comment:
Oh, sorry I didn't notice that you're using actual tabs.  My bad!  Will be fixed in next PR.